ui/src/api/contracts.ts

```diff
@@ -659,9 +662,7 @@ export async function fetchStakerPoolData(
       lastPayoutRound = BigInt(stakingPoolGS.lastPayout.value)
     }
 
-    const result = await callGetStakerInfo(staker, stakingPoolClient)
-
-    const stakedInfo = result.return!
+    const stakedInfo = await callGetStakerInfo(staker, stakingPoolClient)!
```

Comment:
Yeah this is way better for read only calls 👍 
We can follow up and migrate the others to use this pattern